server/controllers/RatingsController.js

```diff
@@ -0,0 +1,175 @@
+import jwt from 'jsonwebtoken';
+
+import db from '../database/models';
+
+const { Article, Rating } = db;
+
+/**
+ * @class RatingsController
+ *
+ * @export {class}
+ */
+class RatingsController {
+  /**
+   * @description - Create New Rating
+   * @static
+   *
+   * @param {object} req - HTTP Request
+   * @param {object} res - HTTP Response
+   *
+   * @memberOf RatingsController
+   *
+   * @returns {object} response JSON Object
+   */
+  static createRating(req, res) {
+    const decoded = jwt.verify(req.headers.authorization, process.env.SECRET_KEY);
```

Comment:
@faksam We already have the token decoded in the auth middleware. To get userId you can simply `req.decoded.userId`